### PR TITLE
Best Entry of team Real-DEEL

### PR DESCRIPTION
**Pull request by mostafaelaraby**
Implementation of our best submission to CLVISION workshop which includes:
- Support a baseline entry to Reinforcement learning track
- Our best submission to Supervised learning track including dark experience loss , buffer tricks and bias control.

### CODE REVIEW

File: examples/clcomp21/Real-DEEL/Methods/base_method.py [hunk 1]
```diff
@@ -0,0 +1,803 @@
...
+        self.task = 0
+        self.epoch = 0
+        # smooth loss initialization
+        self.prev_loss = None
+        self.best_smooth_loss = inf
+
+        self.task_type = "RL" if self._is_rl else "SL"
+        self.iteration_name = "step" if self._is_rl else "epoch"
+        # configuration called specifically to each method inheritting base method
+        self._method_specific_configure(setting)
```
Comment (lebrice) [inline examples/clcomp21/Real-DEEL/Methods/base_method.py:144]: Note: Not sure if you do it, but you could instead implement `configure` in your subclasses and do something like this:
```python3
def configure(self, setting: SettingType): 
    super().configure(setting)
    # and put the code from self._method_specific_configure() here instead

```

**Comment by lebrice** [comment]
Closing this in favour of #236
